blaze-bs4/blaze-bs4-sync.js

```diff
@@ -113,6 +113,12 @@ export const BlazeBs4 = {
     load () {
       import './tooltip/tooltip.js'
     },
-
+  },
+  toast: {
+    template: 'toasts',
+    debug: false,
+    async load() {
+      return import('./toast/toast.js');
+    }
```

Comment:
The load function should not be `async` in the `blaze-bs4-sync` file. If you look at the entries above you see the way the sync imports are done. Please change.

Reply:
Oops. Fixed

Reply:
Also please remove the semicolon. If you want to see if everything is complient run the standard linter on the whole package. It will return all issues. It's annoying in the beginning but ion the long run it will save lots of time.